src/jmh/java/org/javarosa/benchmarks/utils/builder/Constants.java

```diff
@@ -0,0 +1,22 @@
+package org.javarosa.benchmarks.utils.builder;
+
+public class Constants {
+
+    public static String EMPTY_STRING = "";
+    public static String DOUBLE_QUOTE = "\"";
+    public static String SPACE = " ";
+    public static String OPEN_TOKEN = "<";
+    public static String CLOSE_TOKEN = ">";
+    public static String FORWARD_SLASH = "/";
+    public static String EQUALS = "=";
+    public static String HTML = "html";
```

Comment:
html, head, body and title should all be namespaced to XHMTL, set here:

https://github.com/eHealthAfrica/javarosa/pull/9/files#diff-b7c496b2a16d54ba74b5222b2aabe4baR13

Probably the easiest way is to just hardcode the namespace here, so that it becomes:

```
public static String HTML = "h:html";
```
etc.